### PR TITLE
Expand README, add usage docs.

### PR DESCRIPTION
**Pull request by dmitrizagidulin**

### CODE REVIEW

**Comment by dmitrizagidulin** [comment]
@dlongley ready to review.

File: README.md [hunk 1]
```diff
@@ -18,12 +34,156 @@ will be released when a browser-compatible implementation becomes available.
...
+  id: keyAgreementKey.id,
+  type: 'X25519KeyAgreementKey2019',
+  publicKeyBase58: keyAgreementKey.publicKeyBase58
+};
+const keyResolver = async () => publicKeyNode; 
+```
+
+```js
+// A more advanced resolver based on DID doc authentication keys
+const keyResolver = async (keyId) => {
```
Comment (dlongley) [inline README.md]:
```suggestion
const keyResolver = async ({id}) => {
```

File: README.md [hunk 2]
```diff
@@ -18,12 +34,156 @@ will be released when a browser-compatible implementation becomes available.
...
+  // Use veres driver to fetch the authn key directly
+  const authKeyPair = await Ed25519KeyPair.from(await veresDriver.get({did: keyId}));
```
Comment (dlongley) [inline README.md]:
```suggestion
  const authKeyPair = await Ed25519KeyPair.from(await veresDriver.get({did: id}));
```

**Review by dlongley — CHANGES_REQUESTED** [review]

**Review by dlongley — APPROVED** [review]
Actually, going to approve conditionally on my comments being applied to avoid slowing this down; trust you to do the rest, @dmitrizagidulin. Thanks!

**Comment by dmitrizagidulin** [comment]
@dlongley Suggestions/comments merged, thanks!

**Pull request merged**